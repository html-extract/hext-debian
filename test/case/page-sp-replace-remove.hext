<p @text:replace(/\d+/, ""):nonum />
