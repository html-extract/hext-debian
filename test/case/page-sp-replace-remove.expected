{"nonum":" bottles of beer on the wall,  bottles of beer on the wall"}
